scripts/docker-as-drupal

```diff
@@ -898,6 +911,125 @@ elif [ "$1" = "behat" ]; then
   # exit
   exit $behat_exit
 
+#
+# RUN NIGHTWATCH TESTS
+#
+elif [ "$1" = "nightwatch" ]; then
+  shift
+
+  SKIP_DEPENDENCIES=1
+  SKIP_STYLEGUIDE=0
+  SKIP_DB_RESET=0
+  SKIP_DEFAULT_CONTENT=0
+  NIGHTWATCH_GROUPS=()
+  NIGHTWATCH_ARGV=()
+
+  while [ $# -gt 0 ]; do
+    case "$1" in
+      --with-dependencies)
+        SKIP_DEPENDENCIES=0
+        ;;
+      --skip-styleguide-build)
+        SKIP_STYLEGUIDE=1
+        ;;
+      --skip-db-reset)
+        SKIP_DB_RESET=1
+        ;;
+      --skip-default-content)
+        SKIP_DEFAULT_CONTENT=1
+        ;;
+      --group=*)
+        NIGHTWATCH_GROUPS+=("--tag ${1#*=}")
+        ;;
+      --help)
+        print_help
+        yarn nightwatch --help
+        exit $?
+        ;;
+      --)
+        shift
+        NIGHTWATCH_ARGV=($@)
+        break
+        ;;
+    esac
+    shift
+  done
+
+  # Install dependencies
+  if [ $SKIP_DEPENDENCIES -eq 0 ]; then
+    install_dependencies
+  fi
+
+  # Setup settings & Co.
+  setup
+
+  # Reset database
+  if [ $SKIP_DB_RESET -eq 0 ] || drupal_not_installed; then
+    database_reset
+  fi
+
+  # Load default content
+  if [ $SKIP_DEFAULT_CONTENT -eq 0 ]; then
+    enable_default_content
+  fi
+
+  # Disable modules that must not be loaded for tests
+  drush pmu $TEST_DISABLE_MODULES -y
+
+  # Enable modules that must be loaded for tests
+  drush en $TEST_ENABLE_MODULES -y
+
+  if [ ! -z "$DRUPAL_NIGHTWATCH_CONFIG_SET" ]; then
+    IFS=';' read -r -a configs <<< "$DRUPAL_NIGHTWATCH_CONFIG_SET"
```

Comment:
Why don't use default "new line" as list separator?

Reply:
because IFS don't support more than 1 char (using `\n` will split on `\` & `n`)

Reply:
I don't change it for `$REQUIRED_DIRECTORIES`, just loop on the variable without need to use `read` or change `IFS` value.